packages/redux/src/contents/serverInitialState.ts

```diff
@@ -24,12 +24,14 @@ const serverInitialState: ServerInitialState = ({ model }) => {
   const { searchContentRequests, slug, seoMetadata, subfolder } = model;
 
   const contents = searchContentRequests.reduce((acc, item) => {
-    const {
-      filters: { codes, contentTypeCode },
-      searchResponse,
-    } = item;
+    const { searchResponse } = item;
 
-    const hash = generateContentHash({ codes, contentTypeCode });
+    if (!searchResponse.entries[0]) return acc;
+
+    const hash = generateContentHash({
+      codes: searchResponse.entries[0].code,
```

Comment:
This can be refactored a little bit by saving the value of `searchResponse.entries[0]` to a variable, something like this:

```js
const firstSearchResponseItem = searchResponse.entries[0];

if(!firstSearchResponseIItem) {
   return acc;
}

const hash = generateContentHash({
      codes: firstSearchResponseIItem.code,
      contentTypeCode: firstSearchResponseItem.contentTypeCode,
 });


```

Reply:
Resolved ✅